tests/css_check_test.js

```diff
@@ -21,6 +21,17 @@ const getAllocations = async (string) => {
     return response;
 }
 
+const getAssetsJSContent = async () => {
+    const response = await needle('get', `${config.get('PARTICIPANT_URL')}v1/${config.get('ENVIRONMENT_ID')}/${config.get('UID')}/assets.js`).then((res)=>{
+        if (res.statusCode == 200){
+           return res.body;
+        }else throw new Error('Something is wrong: '+res);
```

Comment:
Nitpick: Generally it is prudent to wrap a block with braces even if it can be written in one line without them. Omitting the braces can lead to confusion in the future if another developer add more lines to a block but misses that the block was not wrapped.
```suggestion
        } else {
            throw new Error('Something is wrong: '+res);
        }
```

Reply:
fixed